### DISCUSSION TITLE
Deleting and moving repository to IBAHCM

### DISCUSSION THREAD

**Discussion by JumaMinya** [IBAHCM/RPiR#428] [Archive] [answered]
@IBAHCM/rpir-instructors 
I have just done the instructions, kindly proceed with granting access as directed

**Comment by soniamitchell** [comment, marked as answer]
Hey @JumaMinya, it looks like you've retained write access, which is weird but good. Let us know if you have any other problems.

**Reply by JumaMinya** [reply]
Thanks
